src/guides/about-multiprotocol.html

```diff
@@ -4,14 +4,31 @@
 ---
 
 <div class="container">
+  <h2>Update on multiprotocol</h2>
+  <p><b>Multiprotocol will remain an experimental feature and is not recommended for use.</b>
+  </p>
+  <p>
+      <span>
+          When we launched Home Assistant Yellow, we announced our intent to release firmware supporting multiprotocol, which allows the device's Silicon Labs chip to connect to both Zigbee and Thread networks with one radio.
+      </span>
+  </p>
+  <p>
+      <span>
+          This experimental firmware has been available since December 2022. Through extensive testing, we have found that although it works in some circumstances, it has technical limitations that lead to a worse user experience. We now do not recommend using this firmware, and it will be experimental for the foreseeable future. Instead, we will focus on making sure the dedicated Zigbee and Thread firmwares for Home Assistant Yellow deliver the best experience to users.
```

Comment:
```suggestion
          This experimental firmware has been available since December 2022. Through extensive testing, we have found that although it works in some circumstances, it has technical limitations that lead to a worse user experience. We now do not recommend using this firmware, and it will be experimental for the foreseeable future. Instead, we will focus on making sure the dedicated Zigbee and Thread firmware for Home Assistant Yellow delivers the best experience to users.
```